composer.json

```diff
@@ -24,8 +24,9 @@
         "issues": "https://github.com/RusticiSoftware/TinCanPHP/issues"
     },
     "require": {
-        "php": "~5.4",
+        "php": ">=5.4",
         "ext-openssl": "*",
+        "ext-curl": "*",
```

Comment:
Purely for consistency across my environment, travis and @johnpblochs testing environment. Where the current stream_context* did, didn't and didn't work respectively.

At the time we were testing against 7.0-RC5 and 7.0-RC6 I believe. Now that we have a stable release I'll add reverting the CURL implementation to the list.